experiments/yarn/src/main/scala/org/apache/gearpump/experiments/yarn/master/AmActor.scala

```diff
@@ -97,19 +141,15 @@ class AmActor(appConfig: AppConfig, yarnConf: YarnConfiguration) extends Actor {
       
     case containerRequest: ContainerRequestMessage =>
       LOG.info("AM: Received ContainerRequestMessage")
-      amRMClient ! containerRequest
+      rmClientActor ! containerRequest
```

Comment:
Can we add a comment here that 
What is the response of the ContainerRequest? How the response is passed back? And Who has the responsiblity to handle that response?

From  the API, I  think the Response should be handled here.